goalie/go_mesh_seq.py

```diff
@@ -243,12 +250,16 @@ def indicate_errors(
         :kwarg indicator_fn: function which maps the form, adjoint error and enriched
             space(s) as arguments to the error indicator
             :class:`firedrake.function.Function`
+        :kwarg compute_gradient: if ``True``, the gradient of the QoI with respect to
+            the initial conditions is computed and is available via the `gradient`
+            attribute
+        :type compute_gradient: :class:`bool`
         :returns: solution and indicator data objects
         :rtype1: :class:`~.AdjointSolutionData`
         :rtype2: :class:`~.IndicatorData`
         """
         solver_kwargs = solver_kwargs or {}
-        default_enrichment_kwargs = {"enrichment_method": "p", "num_enrichments": 1}
+        default_enrichment_kwargs = {"enrichment_method": "h", "num_enrichments": 1}
```

Comment:
I feel like p-enrichment is somewhat problematic as a default. There's no guarantee that a p-enrichment will be stable, whereas a h-enriched space should be fine so long as the base space is stable.